README.md

```diff
@@ -1 +1,161 @@
+	* https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
+
+3. Deploy the services  
+First navigate to the folder where you have downloaded the files to and then run one of the following commands, depending on whether you are using Docker Compose or Docker Stack:
+
+    <table>
+      <tr>
+        <td><b>Docker Compose</b></td>
+        <td>
+			docker-compose up  <small>(docker-compose defaults to use a file called <i><b>docker-compose.yml</b></i>)</small><br />
+			docker-compose -f docker-compose.yml up <small>(An alternative filename can be provided using the <i><b>-f</b></i> flag</small> <br />
+			docker-compose -f docker-compose.yml up -d <small>(<i><b>-d</b></i> flag is for "detached mode" i.e. run containers in the background)</small>
+		</td>
+      </tr>
+      <tr>
+        <td><b>Docker Stack</b></td>
+        <td>docker stack deploy --compose-file=docker-compose.yml auditservicedemo</td>
+      </tr>
+    </table>
+
+4. Check the Health of Elasticsearch  
+The health of the Elasticsearch container and / or cluster can be inspected by issuing the following command:    
+	`curl http://127.0.0.1:9200/_cat/health`    
+	`1493041686 13:48:06 docker-cluster green 2 2 14 7 0 0 0 0 - 100.0%`
+
+5. Index a simple customer document 
+	`curl -XPUT 'localhost:9200/customer/external/1?pretty&pretty' -H 'Content-Type: application/json' -d'
+	{
+  		"name": "John Doe"
+	}'`
+    
+	Reponse:
+    
+	`{
+         "_index" : "customer",
+         "_type" : "external",
+         "_id" : "1",
+         "_version" : 1,
+         "result" : "created",
+         "_shards" : {
+             "total" : 2,
+             "successful" : 2,
+             "failed" : 0
+         },
+         "created" : true
+    }`
+
+6. Retrieve a simple customer document  
+	`curl -XGET 'localhost:9200/customer/external/1?pretty&pretty'`  
+  
+	Response:  
+  
+	`{
+	  "_index" : "customer",
+	  "_type" : "external",
+	  "_id" : "1",
+	  "_version" : 1,
+	  "found" : true,
+	  "_source" : { "name": "John Doe" }
+	}`
+
+### Troubleshooting
+
+#### Errors during Start Up
+
+If the following error appears during start up and the Elasticsearch container shuts down shortly after start up:
+> elasticsearch_1  | ERROR: bootstrap checks failed  
+> elasticsearch_1  | max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
+
+This error can be resolved by issuing the following command on the Linux host:  
+* `sudo sysctl -w vm.max_map_count=262144`
+
+More information regarding `vm.max_map_count` can be found in the Elasticsearch official Documentation [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode) and [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html).
+
+#### Warnings during Start Up
+
+If the following warning is seen during start up it can be disregarded:
```

Comment:
If this warning can be disregarded then is there any setting that could be applied to turn it off?